spec/features/admin/navigation_spec.rb

```diff
@@ -19,15 +19,20 @@
       end
     end
 
-    it "404 errors when accessing cart/merchant" do
+    it "404 errors when accessing cart" do
       admin = User.create(name: "David", address: "123 Test St", city: "Denver", state: "CO", zip: "80204", email: "123@example.com", password: "password", role: 3)
       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
 
-      visit "/merchant"
+      visit "/cart"
 
       expect(page).to have_content("The page you were looking for doesn't exist.")
+    end
 
-      visit "/cart"
+    it "404 errors when accessing merchant paths" do
+      admin = User.create(name: "David", address: "123 Test St", city: "Denver", state: "CO", zip: "80204", email: "123@example.com", password: "password", role: 3)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+      visit "/merchant"
```

Comment:
As it turns out, it wasn't necessary to break these up into two separate tests - I was just having an issue with a `while` loop so it was infinitely trying to render the 404 page.